dev_cz.jl

```diff
@@ -0,0 +1,285 @@
+using Pkg; Pkg.activate(".")
```

Comment:
Whats this file yo

Reply:
Boomer

Reply:
Sneaky bonus code :) just for you guys ... no that one shouldn't be there.